distribution/docs/src/main/resources/content/_running/starting-intro.adoc

```diff
@@ -51,6 +51,15 @@ Comment out the -XX:+DisableAttachMechanism line and re-number remainder lines a
 ----
 ====
 
+.Solr Memory Considerations
+[NOTE]
+====
+From the https://lucene.apache.org/solr/guide/7_0/taking-solr-to-production.html[Solr Documentation]
+----
+For production, you’ll want to increase the maximum heap size based on the memory requirements of your search application; values between 10 and 20 gigabytes are not uncommon for production servers.
```

Comment:
This wording sounds a bit too casual considering the style of our documentation. 
`For production, increase the maximum heap size based on....`

Reply:
This is straight excerpt from solr docs

Reply:
👼

Reply:
Go back to army :trollface:

Reply:
The quote will also be in a box when rendered emphasising that the information comes from the Solr Docs linked above it.

Reply:
10-20 GB heap? I would think that would bring most JVM GCs to their knees.

Reply:
I feel this information does not belong in a quick start. It does not tell the user to take action, which something in a quick start should do.